packages/storefront-lwc/src/modules/commerce/productAvailability/productAvailability.scss

```diff
@@ -1,3 +1,15 @@
-.availability {
-  margin-top: 1.071em;
+.pdp .availability {
+    margin-top: 1.071em;
 }
+
+.basket .availability {
+    margin-top: 0;
+    font-size: 1rem;
+}
+.basket .availability-label {
+    display: none;
+}
+
+.basket .availability-message {
```

Comment:
Would these CSS apply if the parent component does not have the .basket class? We should keep the CSS specific to the component it is used for.

Reply:
This component is being used on both the PDP and the Basket, we need two different styles for these two places. That's why I added a context to distinguish them.

Reply:
So maybe we should leave a comment in the code about what effects the context has (since it is connected with CSS styling). If someone passes in something other than basket (thinking about reusability) then things will not work the same way for them.

However, i am of the opinion that we should not be trying to make one component work for several different scenarios. But that could be a wider discussion on making components. @blittle what do you think?

Reply:
okay, I added code comments.